src/events/api/events.ts

```diff
@@ -47,6 +54,33 @@ export const getAttendanceEvent = async (id: number): Promise<IAttendanceEvent>
   return attendanceEvent;
 };
 
+export const getEventPayment = async (event_id: number): Promise<IPayment> => {
+  try {
+    const ret = await get<IPayment>(`${ATTENDANCE_EVENT_API_URL}${event_id}${EVENT_PAYMENT_URL}`);
+    return ret;
+  } catch (response) {
+    throw new Error('Kunne ikke hente betalingsinformasjon for arrangementet!');
+  }
+};

```

Comment:
I think this has also been implemented as part of the `payments` slice in the payments app